tradeexecutor/analysis/trade_analyser.py

```diff
@@ -433,6 +451,11 @@ def calculate_summary_statistics(self) -> TradeSummary:
 
         duration = datetime.timedelta(0)
 
+        average_winning_trade = []
+        average_losing_trade = []
+        average_winning_trade_duration = []
+        average_losing_trade_duration = []
```

Comment:
Since these are all lists containing data points, would it make more sense to name them differently?

Because they do not actually represent the "average winning trade" and "average duration", this is computed later based on the data points in these lists.

Reply:
Yes, I should name them differently.